src/ui-components/empty-state/empty-state.component.tsx

```diff
@@ -19,7 +19,7 @@ export default function EmptyState(props: EmptyStateProps) {
         <p className="omrs-type-body-regular">
           <Trans
             i18nKey="emptyStateText"
-            values={{ displayText: props.displayText }}
+            values={{ displayText: props.displayText.toLowerCase() }}
```

Comment:
This works fine. The string comes in already translated so what I'm doing here is ensuring that it is set to lowercase. Do you still feel a text-transform would be better suited for this?

Reply:
This is also fine IMO :+1:

Reply:
Great, thanks Brandon